packages/react/src/avatar-context/AvatarContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { type ComponentPropsWithoutRef, createContext } from "react";
+
+import type { Avatar } from "../avatar";
+
+type AvatarContextType = {
+  size: ComponentPropsWithoutRef<typeof Avatar>["size"];
+};
+
+export const AvatarContext = createContext<{
```

Comment:
let's call this `AvatarGroupContext`
```suggestion
export const AvatarGroupContext = createContext<{
```